apprise/Apprise.py

```diff
@@ -369,91 +366,118 @@ def notify(self, body, title='', notify_type=common.NotifyType.INFO,
         such as turning a \n into an actual new line, etc.
         """
 
-        return py3compat.asyncio.tosync(
-            self.async_notify(
+        try:
+            # Process arguments and build synchronous and asynchronous calls
+            # (this step can throw internal errors).
+            sync_partials, async_cors = self._create_notify_calls(
                 body, title,
                 notify_type=notify_type, body_format=body_format,
                 tag=tag, match_always=match_always, attach=attach,
-                interpret_escapes=interpret_escapes,
-            ),
-            debug=self.debug
```

Comment:
does the new `async` functions you set up still pass the `debug` flag as True if set globally in the Apprise object as it was here? Not sure if there is value here or not; i was thinking for troubleshooting, doesn't the `debug` on internal async calls increase the verbosity of some actions? Or no?

Reply:
Yes; the debug flag is set [here](https://github.com/caronc/apprise/blob/3d1f431c74f2b58256360d6592a524c2c6460002/apprise/Apprise.py#L404) for new event loops and [here](https://github.com/caronc/apprise/blob/3d1f431c74f2b58256360d6592a524c2c6460002/apprise/Apprise.py#L415) for existing event loops. And it looks like this mode does still offer some [useful features](https://docs.python.org/3/library/asyncio-dev.html#asyncio-debug-mode). :cool: